drivers/sensor/ti/tmp435/tmp435.c

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright (c) 2024 Bittium Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT ti_tmp435
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/logging/log.h>
+#include "tmp435.h"
+
+LOG_MODULE_REGISTER(TMP435, CONFIG_SENSOR_LOG_LEVEL);
+
+static inline int tmp435_reg_read(const struct tmp435_config *cfg, uint8_t reg, uint8_t *buf,
+				  uint32_t size)
+{
+	return i2c_burst_read_dt(&cfg->i2c, reg, buf, size);
+}
+
+static inline int tmp435_reg_write(const struct tmp435_config *cfg, uint8_t reg, uint8_t *buf,
+				   uint32_t size)
+{
+	return i2c_burst_write_dt(&cfg->i2c, reg, buf, size);
+}
+
+void tmp435_get_status(const struct tmp435_config *cfg, uint8_t *status)
```

Comment:
Missing static. Also should probably be inline like the previous very short functions

Reply:
Done